crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -553,6 +553,93 @@ impl<'w, 's, T: Component> Fetch<'w, 's> for WriteFetch<T> {
     }
 }
 
+pub struct Has<T: Component>(PhantomData<T>);
```

Comment:
Why not `Has<Q: WorldQuery>`, having to do `(Has<T>, Has<U>)` and then `&&` the two bools seems annoying